java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -741,7 +956,8 @@ private void handleInTransitClick() {
     }
 
     private void handleTrainSelectionChanged() {
-        if (!trainsFromButtonGroup.getSelection().getActionCommand().equals("TRAINSFROMOPS")) {
```

Comment:
Revert this. The selected button cannot be null at this point. It is initialized to one of the four buttons during form creation. If it does become null then its aproblem that needs to be fixed, not ignored. The default is TRAINSFROMSETLATER